[2024-06-30 19:01:09,907] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T04:00:00+00:00 [queued]>
[2024-06-30 19:01:10,057] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T04:00:00+00:00 [queued]>
[2024-06-30 19:01:10,057] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 19:01:10,057] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-06-30 19:01:10,058] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-06-30 19:01:10,215] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_datalake_table> on 2024-06-30 04:00:00+00:00
[2024-06-30 19:01:10,396] {standard_task_runner.py:52} INFO - Started process 1308 to run task
[2024-06-30 19:01:10,661] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_datalake_dag', 'create_datalake_table', 'scheduled__2024-06-30T04:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpj9s48u99', '--error-file', '/tmp/tmpk1o4rein']
[2024-06-30 19:01:11,099] {standard_task_runner.py:80} INFO - Job 12: Subtask create_datalake_table
[2024-06-30 19:01:13,700] {task_command.py:371} INFO - Running <TaskInstance: example_datalake_dag.create_datalake_table scheduled__2024-06-30T04:00:00+00:00 [running]> on host 12d6cc628018
[2024-06-30 19:01:15,450] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_datalake_dag
AIRFLOW_CTX_TASK_ID=create_datalake_table
AIRFLOW_CTX_EXECUTION_DATE=2024-06-30T04:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-30T04:00:00+00:00
[2024-06-30 19:01:15,593] {base.py:68} INFO - Using connection ID 'datalake_postgres' for task execution.
[2024-06-30 19:01:15,762] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS datalake_table (
            id SERIAL PRIMARY KEY,
            data VARCHAR(50)
        );
        , parameters: None
[2024-06-30 19:01:15,773] {postgres.py:94} INFO - NOTICE:  relation "datalake_table" already exists, skipping

[2024-06-30 19:01:17,373] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=example_datalake_dag, task_id=create_datalake_table, execution_date=20240630T040000, start_date=20240630T190109, end_date=20240630T190117
[2024-06-30 19:01:18,218] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-06-30 19:01:19,159] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
